server/seeder.js

```diff
@@ -1,5 +1,5 @@
 const faker = require('faker');
-const {db, Reviews} = require('./db.js'); // commented out to stop jest from running full code
+const {db, Reviews} = require('./db.js');
```

Comment:
I am not sure what is better practice to import all exports from another file into single object or multiple.
for ex. const x = require('./db.js') and access x.db and x.Reviews. OR to import them separately?

Reply:
Good point. I will look into this, isn't this destructuring?